include/openssl/ecdsa.h

```diff
@@ -232,5 +232,6 @@ BSSL_NAMESPACE_END
 #define ECDSA_R_NOT_IMPLEMENTED 103
 #define ECDSA_R_RANDOM_NUMBER_GENERATION_FAILED 104
 #define ECDSA_R_ENCODE_ERROR 105
+#define ECDSA_R_MISMATCHED_SIGNATURE 106
```

Comment:
Maybe we should reserve some space for openssl/boringssl defined values here. Offset of 100 reasonable?